csrc/scheduler/mma_utils.cpp

```diff
@@ -281,6 +281,143 @@ void scheduleWarpTileWithReduction(TensorView* tv, MatMulTileOptions tile) {
   // [Kwo Mwo Nwo Mw Nw Mi Ni Ki]
 }
 
+void scheduleWarpTile(
```

Comment:
Actually, is this still used after the removal of `scheduleLdMatrixLoads`?

Reply:
You're right. I reverted a change that was using this. Now we don't use it. We might use it in the future but I'll remove it until then.